apps/examples/testcase/le_tc/tcp_tls/tcp_tls_stress.c

```diff
@@ -212,13 +212,13 @@ tcp_tls_result_e tcp_tls_config_get(tls_config * config)
 
 static void wm_get_apinfo(wifi_manager_ap_config_s *apconfig)
 {
-	strncpy(apconfig->ssid, WM_AP_SSID, WIFIMGR_SSID_LEN-1);
-	apconfig->ssid[WIFIMGR_SSID_LEN-1] = '\0';
+	strncpy(apconfig->ssid, WM_AP_SSID, strlen(WM_AP_SSID));
```

Comment:
I agree with you. It would be better to use 'sizeof' and remove the below line (216), because WM_AP_SSID and WM_AP_PASSWORD are defined variables by the configuration. Modification will be applied. Thanks.